src/components/PetDetail.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PetThumbnail from "./PetThumbnail";
+import PetAttributesSection from "./PetAttributesSection";
+
+const PetDetail = props => {
+  const { imageUrl, imageUrlAltText, aboutMe, attributes, animalName } = props;
+  return (
+    <div className="container">
```

Comment:
not sure where, but i suspect the acmp number should be somewhere on here.

Reply:
@martypowell as a key or id?

Reply:
actually displayed. From what i undersatand a person would reference that number in the case that there are a bunch of brunos.

Reply:
The prod page has his included in the attributes section so Ive added ID to that for now. Will have to verify with Sam if this is okay.

Reply:
I think this might go in a different location, but this works for now.